vendor/key_gen/src/main.rs

```diff
@@ -5,58 +5,65 @@ use bitcoin_support::{
 use ethereum_support::ToEthereumAddress;
 use rand::OsRng;
 use secp256k1_support::{KeyPair, SecretKey, SECP};
+#[macro_use]
+extern crate log;
+extern crate pretty_env_logger;
+use std::env;
 
 fn main() {
+    env::set_var("RUST_LOG", "info");
```

Comment:
🔧 should we create an issue to move this it's own "utils" repo?

Reply:
Done in #626